report/patient_vara_findings.fodt

```diff
@@ -147,35 +148,37 @@
   </office:script>
  </office:scripts>
  <office:font-face-decls>
-  <style:font-face style:name="Andale Sans UI" svg:font-family="&apos;Andale Sans UI&apos;" style:font-family-generic="system" style:font-pitch="variable"/>
-  <style:font-face style:name="Arial" svg:font-family="Arial"/>
-  <style:font-face style:name="Arial1" svg:font-family="Arial" style:font-family-generic="swiss"/>
-  <style:font-face style:name="DejaVu Sans" svg:font-family="&apos;DejaVu Sans&apos;" style:font-family-generic="system" style:font-pitch="variable"/>
+  <style:font-face style:name="Arial1" svg:font-family="Arial"/>
```

Comment:
Was this font change intentional?

Reply:
There are actually a lot of font changes, but a lot of them are from `Arial1` to `Arial`, so it seems like Arial is the intended font.

Reply:
The template was changed according to how Hannes wanted it to look like. These changes are done by LibreOffice and not directly on the text file.